pkg/generator/generator.go

```diff
@@ -215,15 +223,16 @@ func (g Generator) GenerateConfig(ctx context.Context, installation, app, namesp
 		Name:      name,
 		Namespace: namespace,
 		Labels: map[string]string{
-			key.KubernetesManagedByLabel: "Helm",
-			key.GiantswarmManagedByLabel: project.Name(),
-			key.ConfigVersion:            ref,
-			key.AppLabel:                 name,
+			annotation.ConfigMajorVersion: ref,
```

Comment:
That doesn't look correct. It should be a number without the `v` prefix.

Reply:
Maybe I misunderstood the purpose of the flag then.
What's the label we set on AppCR to set config version then?

Reply:
`config.giantswarm.io/major-version: "3"` - does that answer your question? :D